src/Stryker.CLI/Stryker.CLI/OptionsBuilder.cs

```diff
@@ -40,10 +42,20 @@ public class OptionsBuilder
                         .Build().GetSection("stryker-config");
             }
 
+            var testProjectName = testProjectNameFilter.Value();
+            if (testProjectNameFilter.HasValue())
```

Comment:
Is this neccacary? If we just pass the filter as it is we can later determine during the run if the filter should be used and how it should be used. I don't think the `OptionsBuilder` should be responsible for modifying the value.

Reply:
testProjectNameFilter.Value() should simply be passed to GetOption() and handled just like all other options, agreed.